### PR TITLE
Rename Users documentation to Manual

### PR DESCRIPTION
**Pull request by Simran-B**
@dothebart: do the changes to both `.gitignore` files look good to you?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,14 +51,6 @@ lib/CMakeFiles/
 lib/cmake_install.cmake
 
 Documentation/Examples/*.generated
-Documentation/Books/Users/book.json
```
Comment (pluma) [inline .gitignore]: I suppose book.json is some build artefact? Is it no longer being generated? It's missing in the Books/.gitignore file below

Comment (Simran-B) [inline .gitignore]: From what source file would it be generated?! It contains a title and description... I'm actually baffled how it made it onto the ignore list.

**Comment by dothebart** [comment]
the different scripts generate it with different plugins (i.e. for the website you want the piwik plugin)

**Comment by Simran-B** [comment]
Fast-forwarded changes.

**Pull request closed without merging**